### PR TITLE
Update annotations

### PR DESCRIPTION
**Pull request by birdage**
requires https://github.com/asascience-open/ooi-ui-services/pull/361

### CODE REVIEW

**Comment by birdage** [comment]
@FBRTMaka can you take a nice hard look at  `ooiui/templates/science/plotting.html` to make sure everything is ok

**Comment by birdage** [comment]
@FBRTMaka also doing some tidy up now

**Comment by DanielJMaher** [comment]
@birdage Call when it is ready Andy.  @oceanzus Can you help review?

**Comment by FBRTMaka** [comment]
@birdage, ok will do!

File: ooiui/static/css/common/toc_menu.css [hunk 1]
```diff
@@ -23,6 +23,19 @@ label {
...
+}
+
+.nav .nav-list .tree li a:active{
+    background-color:lightblue;
+}
+
+.nav .nav-list .tree .active-toc-item{
+    background-color:lightblue;
+}
+
```
Comment (FBRTMaka) [inline ooiui/static/css/common/toc_menu.css]: @birdage , does this do anything funky with the nav bar at the top?

File: ooiui/templates/science/plotting.html [hunk 2]
```diff
@@ -469,44 +433,51 @@
         }
       }
     });
+    
+    this.listenTo(self.collections.annotations, 'reset', function(collection) {
+      console.log(collection,"annotations are here!!!!!")
```
Comment (FBRTMaka) [inline ooiui/templates/science/plotting.html]: @birdage , YAYYYY (but needed?) :P

**Comment by FBRTMaka** [comment]
@birdage looks good to me! will merge when you're ready

**Comment by FBRTMaka** [comment]
MERGING!!!!

**Comment by DanielJMaher** [comment]
@FBRTMaka @birdage The abck end side is failing the tests.  We donthave to merge this until that is fixed.

**Comment by DanielJMaher** [comment]
Whoops, too late.

**Comment by FBRTMaka** [comment]
OOPS!

**Pull request merged**